### PR TITLE
change xds version and increment chart version

### PR DESCRIPTION
**Pull request by moosecanswim**
I updated the xds docker image tag and incremented the chart

### CODE REVIEW

**Comment by tilleryd** [comment]
@chrisbsmith Shouldn't this go into `master` like how we are merging PRs into `gm-2`? We should only merge into release branches as a side effect of merging into another long running branch, in this case master. We can cherry pick commits onto release branches...

**Comment by moosecanswim** [comment]
https://jenkins.development.deciphernow.com/job/helm-chart/job/helm-charts/view/change-requests/job/PR-252/

**Comment by moosecanswim** [comment]
This passed but it did not hit github

File: xds/values.yaml [hunk 1]
```diff
@@ -14,7 +14,7 @@ global:
 xds:
   image: 'docker.production.deciphernow.com/deciphernow/gm-xds:0.2.6'
```
Comment (tilleryd) [inline xds/values.yaml]: Shouldn't the Docker image tag be 0.2.10 as well?

**Comment by tilleryd** [comment]
@chrisbsmith correct on merging into the release branches

**Comment by chrisbsmith** [comment]
@moosecanswim this PR has also become `stale`.  And since we aren't using `xds` anymore, I don't think it's necessary to merge.  I will close this one by the end of the day unless I hear otherwise from you.

**Pull request closed without merging**